cluster/gce/util.sh

```diff
@@ -2455,24 +2470,25 @@ function create-subnetworks() {
 
   # Look for the alias subnet, it must exist and have a secondary
   # range configured.
-  local subnet=$(gcloud compute networks subnets describe \
+  local subnet
+  subnet=$(gcloud compute networks subnets describe \
     --project "${NETWORK_PROJECT}" \
-    --region ${REGION} \
-    ${IP_ALIAS_SUBNETWORK} 2>/dev/null)
-  if [[ -z ${subnet} ]]; then
+    --region "${REGION}" \
+    "${IP_ALIAS_SUBNETWORK}" 2>/dev/null || true)
```

Comment:
Should not be there. Thanks for spotting.

Reply:
heh, the `|| true` here is here because we store the value and then check if empty below (and then create the subnet if so), we don't want to fail if the subnet doesn't exist yet.